oss_test.go

```diff
@@ -42,196 +50,196 @@ func TestStorage(t *testing.T) {
+				length, err := driver.Size(rootFolder + "Get/1.txt")
 				assert.Nil(t, err)
 				assert.Equal(t, int64(7), length)
-				assert.Nil(t, driver.DeleteDirectory("Get"))
+				assert.Nil(t, driver.DeleteDirectory(rootFolder+"Get"))
 			},
 		},
 		{
 			name: "LastModified",
 			setup: func() {
-				assert.Nil(t, driver.Put("LastModified/1.txt", "Goravel"))
-				assert.True(t, driver.Exists("LastModified/1.txt"))
-				date, err := driver.LastModified("LastModified/1.txt")
+				assert.Nil(t, driver.Put(rootFolder+"LastModified/1.txt", "Goravel"))
+				assert.True(t, driver.Exists(rootFolder+"LastModified/1.txt"))
+				date, err := driver.LastModified(rootFolder + "LastModified/1.txt")
 				assert.Nil(t, err)
 
 				l, err := time.LoadLocation("UTC")
 				assert.Nil(t, err)
-				assert.Equal(t, time.Now().In(l).Format("2006-01-02 15"), date.Format("2006-01-02 15"))
-				assert.Nil(t, driver.DeleteDirectory("LastModified"))
+				assert.Equal(t, carbon.Now().ToStdTime().In(l).Format("2006-01-02 15"), date.Format("2006-01-02 15"))
+				assert.Nil(t, driver.DeleteDirectory(rootFolder+"LastModified"))
 			},
 		},
 		{
 			name: "MakeDirectory",
 			setup: func() {
-				assert.Nil(t, driver.MakeDirectory("MakeDirectory1/"))
-				assert.Nil(t, driver.MakeDirectory("MakeDirectory2"))
-				assert.Nil(t, driver.MakeDirectory("MakeDirectory3/MakeDirectory4"))
-				assert.Nil(t, driver.DeleteDirectory("MakeDirectory1"))
-				assert.Nil(t, driver.DeleteDirectory("MakeDirectory2"))
-				assert.Nil(t, driver.DeleteDirectory("MakeDirectory3"))
-				assert.Nil(t, driver.DeleteDirectory("MakeDirectory4"))
+				assert.Nil(t, driver.MakeDirectory(rootFolder+"MakeDirectory1/"))
+				assert.Nil(t, driver.MakeDirectory(rootFolder+"MakeDirectory2"))
+				assert.Nil(t, driver.MakeDirectory(rootFolder+"MakeDirectory3/MakeDirectory4"))
+				assert.Nil(t, driver.DeleteDirectory(rootFolder+"MakeDirectory1"))
+				assert.Nil(t, driver.DeleteDirectory(rootFolder+"MakeDirectory2"))
+				assert.Nil(t, driver.DeleteDirectory(rootFolder+"MakeDirectory3"))
+				assert.Nil(t, driver.DeleteDirectory(rootFolder+"MakeDirectory4"))
 			},
 		},
 		{
 			name: "MimeType",
 			setup: func() {
-				assert.Nil(t, driver.Put("MimeType/1.txt", "Goravel"))
-				assert.True(t, driver.Exists("MimeType/1.txt"))
-				mimeType, err := driver.MimeType("MimeType/1.txt")
+				assert.Nil(t, driver.Put(rootFolder+"MimeType/1.txt", "Goravel"))
+				assert.True(t, driver.Exists(rootFolder+"MimeType/1.txt"))
+				mimeType, err := driver.MimeType(rootFolder + "MimeType/1.txt")
 				assert.Nil(t, err)
 				mediaType, _, err := mime.ParseMediaType(mimeType)
 				assert.Nil(t, err)
 				assert.Equal(t, "text/plain", mediaType)
 
 				fileInfo := &File{path: "logo.png"}
-				path, err := driver.PutFile("MimeType", fileInfo)
+				path, err := driver.PutFile(rootFolder+"MimeType", fileInfo)
 				assert.Nil(t, err)
 				assert.True(t, driver.Exists(path))
 				mimeType, err = driver.MimeType(path)
```

Comment:
Shouldn't we delete this folder after use?
```suggestion
			        mimeType, err = driver.MimeType(path)
                                assert.Nil(t, driver.DeleteDirectory(rootFolder+"MimeType"))
```